Octokit.Reactive/Clients/IObservableMigrationClient.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Octokit.Reactive
+{
+    public interface IObservableMigrationClient
+    {
+        /// <summary>
+        /// A client for GitHub's Migrations API
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/migration/migrations/">Enterprise License API documentation</a> for more information.
+        /// </remarks>
+        IObservableMigrationsClient Migration { get; }
```

Comment:
Will do it, but I've also mentioned it in #1207 as this is a special case caused by the API docs.